intro_to_python/intro_to_python.md

```diff
@@ -188,84 +196,88 @@ If you are interested in using Python for data analysis, you might want to write
+  </div>
+
+* You can also download [just the latest version of Python directly at python.org](https://www.python.org/downloads/). If you also want Jupyter or any IDEs, you will need to download those separately as well. [You can install Jupyter here](https://jupyter.org/install) and [here is a list of IDEs where you can write Python code](https://wiki.python.org/moin/IntegratedDevelopmentEnvironments).
+
+  <div class = "help">
+  That list of IDEs might seem a bit intimidating! Let's narrow things down to a few good options. If you don't know where to start, [Thonney](https://thonny.org/) is a good option for beginners, and actually comes with Python built in. [Spyder](https://www.spyder-ide.org/) was designed for scientific programming, and is a popular option for data scientists and analysts. [PyCharm](https://www.jetbrains.com/pycharm/) is popular with professional developers and has many features that other IDEs lack, but may come with a steeper learning curve. Both Spyder and PyCharm come with the Anaconda distribution.
+  </div>
+
+Using Python in the Cloud
+=======
 
-### Knowledge Check 4
+If you don't want to download Python and would instead prefer to continue practicing in the cloud, [Google Colaboratory](https://colab.research.google.com/), or Google Colab, provides an online Jupyter notebook environment that is completely free (though you do need a Google account). The introduction is actually written as a Colab notebook itself, so you can jump right in practicing. While it won't look exactly like the Jupyter notebook we used for this module, it will be a very similar experience, with a mix of code cells and text cells written in Markdown.
+
+You can even see what this module's notebook looks like in Google Colab! Once you've worked through the introduction above:
+
+1. Click "File" in the upper left-hand corner and then "Open notebook" in the drop-down menu:
+
+   ![Opening a notebook in Google Colab](media/google_colab_open_nb.png)
+
+ 2. In the window that appears, one of the options at the top is **GitHub**. Click that, and you'll see where you can type in a GitHub URL.
+ 3. In that space, copy and paste the repository where these modules live: [https://github.com/arcus/education_modules](https://github.com/arcus/education_modules).
+ 4. Click the magnifying glass to the far right. You should see a list of the Python notebooks in that repository.
+ 5. When you find the one you want, the button on the far right (an upward-slanting arrow in a box) will open the notebook file as a Colab notebook in another tab.
+
+   <div style="display:none">@gifPreload</div>
+
+   <figure>
+     <img src="https://github.com/arcus/education_modules/blob/update-intro-to-python/intro_to_python/media/github_repo_colab.png?raw=true" height="500" width="800" alt="Finding a GitHub repository in Google Colab" data-alt="https://github.com/arcus/education_modules/blob/update-intro-to-python/intro_to_python/media/github_repo_colab.gif?raw=true">
```

Comment:
I'm conflicted here -- since the info is all contained in the text above, this would typically be considered redundant and the alt would then require a blank tag. On the other hand, the fig caption says "click on the image to play the demo", so users w/o visual access are still going to be aware of the presence of an image, and perhaps then having a blank tag could make them wonder if they're missing out on important stuff....

What are your thoughts?